cogs/utils/checks.py

```diff
@@ -30,6 +30,8 @@ def is_owner():
 def check_permissions(ctx, perms):
     if is_owner_check(ctx):
         return True
+    elif not perms:
```

Comment:
`bool({})` is `False`, but if the dictionary contains any keys, regardless of value, `bool(d)` is `True`